style-guide/blog-guidelines.rst

```diff
@@ -0,0 +1,181 @@
+.. _blog-guidelines:
+
+===============
+Blog guidelines
+===============
+
+Developers often ask an Information Developer to edit a blog that they
+have written and placed in the
+https://github.com/rackerlabs/docs-developer-blog repository. Blogs in this
+repository are published to the Rackspace Developer Blog site at
+https://developer.rackspace.com/blog/.
+
+When you write a blog, use the writing guidelines in this Style Guide as you
+would with other documentation that is curated by the Information Development
+team. Start with the information in the :ref:`quickstart`.
+
+This topic provides additional guidelines specific to writing blogs for the
+Rackspace Developer Blog site.
+
+
+
+Things to consider before writing a blog
+----------------------------------------
+
+Before you start writing, consider the following questions:
+
+- What is the goal of this developer blog? Describe it in two sentences or
+  less.
+
+  - Is it to be helpful and answer a question or offer a How-To?
+  - Is it to provide “thought leadership” or show our expertise in an area?
+    If so, describe what you’d like to convey.
+  - Is it something else altogether? If so, what? It’s possible that your
+    idea might be better presented as an article in the How-To or as a
+    technical or marketing white paper.
```

Comment:
Ooh we should probably nix the white paper suggestion, given that we don't have them anymore.

Reply:
fixed

Reply:
" ... that idea might be better presented as an article on the How-To support site ..."

Reply:
fixed